raft/raft.go

```diff
@@ -1039,7 +1039,7 @@ func stepLeader(r *raft, m pb.Message) error {
 		pr.RecentActive = true
 
 		if m.Reject {
-			r.logger.Debugf("%x received msgApp rejection(lastindex: %d) from %x for index %d",
+			r.logger.Debugf("%x received MsgAppResp rejection(lastindex: %d) from %x for index %d",
```

Comment:
`MsgApp` makes slightly more sense to me, because it is the `MsgApp` message that was rejected and reported back via `MsgAppResp` message.

Reply:
Thanks for your reply , you're right.
But there's a little doubt that MsgAppResp was recevied not MsgApp.
(reference line 1087: "r.logger.Infof("%x sent MsgTimeoutNow to %x after **received MsgAppResp**", r.id, m.From) ")

Change the log information to “%x received MsgAppResp(MsgApp was rejected， lastindex: %d) from %x for index %d”

Reply:
/cc @jingyih

Reply:
sounds good to me. Thanks!

Reply:
done，please review

Reply:
/cc @jingyih